homeassistant/components/media_player/plex.py

```diff
@@ -259,65 +277,82 @@ def media_content_type(self):
             return MEDIA_TYPE_TVSHOW
         elif media_type == 'movie':
             return MEDIA_TYPE_VIDEO
+        elif media_type == 'track':
+            return MEDIA_TYPE_MUSIC
         return None
 
     @property
     def media_duration(self):
         """Duration of current playing media in seconds."""
         if self.session is not None:
-            return self.session.duration
+            return self._convert_na_to_none(self.session.duration)
 
     @property
     def media_image_url(self):
         """Image url of current playing media."""
         if self.session is not None:
-            return self.session.thumbUrl
+            thumb_url = self._convert_na_to_none(self.session.thumbUrl)
+            if "__NA__" in thumb_url:
```

Comment:
We no longer have to check for `__NA__` since you already convert it to None if it is ?